dashboard/app/models/activity_section.rb

```diff
@@ -51,11 +51,27 @@ def summarize
       duration: duration,
       remarks: remarks,
       description: Services::I18n::CurriculumSyncUtils.get_localized_property(self, :description),
-      tips: tips,
+      tips: localized_tips,
       progressionName: localized_progression_name
     }
   end
 
+  # Translates the content of tips in the adequate format.
+  # @attr [Array<Hash>] tips each Hash contains a "type" and a "markdown". get_localized_property returns the same
+  # [Array<Hash>] when the locale is the default_locale, otherwise, it returns or Array of to translated "markdown"
+  # content [Array<String or NilClass>].
+  #
+  # @return [Array <Hash>] copy of tips containing translated content
+  def localized_tips
+    return nil unless tips
+    all_localized_tips = Services::I18n::CurriculumSyncUtils.get_localized_property(self, :tips)
+    tips_clone = tips.map(&:clone)
+    tips_clone.each_with_index do |tip, index|
+      tip["markdown"] = all_localized_tips[index] unless (all_localized_tips[index] == tip) || all_localized_tips[index].nil?
```

Comment:
When the language is the default language, `get_localized_property` returns the same object which would be and Array of Hash as per [render_translations.rb](https://github.com/code-dot-org/code-dot-org/blob/7207da0f5b73598e500972d0d916150d8f5fe393/dashboard/lib/services/i18n/curriculum_sync_utils/render_translations.rb). In that case `all_localized_tips[index] == tip` will be true, since the `tip` will be a Hash as well as the `all_localized_tips[index]`. The condition `all_localized_tips[index] == tip` could be substituted for `locale==default_locale`